src/host-callback.js

```diff
@@ -57,7 +57,7 @@ class PostMessageCallbackMananger {
    * @param {function(): undefined} callback
    * @return {number} A handle that can used for cancellation.
    */
-  queueCallback(callback) {
+  queueCallback_(callback) {
```

Comment:
these two are taking the JSCompiler interpretation of private as file-level private, not class-level :) Doesn't seem like too much of a stretch?

Reply:
No objections here. IIRC the intent was for this to be internal, so minimizing impact of this sgtm.